compiler/packages/eslint-plugin-react-compiler/package.json

```diff
@@ -26,7 +26,7 @@
     "@types/eslint": "^8.56.12",
     "@types/node": "^20.2.5",
     "babel-jest": "^29.0.3",
-    "eslint": "8.57.0",
+    "eslint-v8": "npm:eslint@^8.57.1",
```

Comment:
The eslint compiler test failures are due to this change.  It's now using v9, which requires different config input for `RuleTester`.  It would be similar to what the hooks plugin tests set up in the v9 tests.
![image](https://github.com/user-attachments/assets/6e5ccef2-d3e4-4400-b202-3736a689eb05)